AnalyticsRallyHere/Source/AnalyticsRallyHere/Private/AnalyticsRallyHere.cpp

```diff
@@ -21,9 +21,8 @@ TSharedPtr<IAnalyticsProvider> FRH_Analytics::CreateAnalyticsProvider(const FAna
 {
 	if (GetConfigValue.IsBound())
 	{
-		Config ConfigValues;
-		ConfigValues.APIKey = GetConfigValue.Execute(Config::GetKeyNameForAPIKey(), true);
-		ConfigValues.APIServer = GetConfigValue.Execute(Config::GetKeyNameForAPIServer(), true);
+		FAnalyticsET::Config ConfigValues;
+		ConfigValues.APIServerET = GetConfigValue.Execute(FAnalyticsET::Config::GetKeyNameForAPIServer(), true);
```

Comment:
Doesn't this mean we are looking for config values with names that contain ET in them, rather than with our RallyHere specific names?

Reply:
It does, but that conforms to the interface.  This becomes circular with needing a new Config class if we want to customize the config, but our config is basically a subset of theirs so that requirement is only for completeness not functionality.  And we would have to still fill in a ET config object and set it (mirroring those values) to conform to a few functions that have implementations on the ET interface.